<!DOCTYPE html>
<html lang="en">

<head>

    <!-- Basic Page Needs
  ================================================== -->
    <meta charset="utf-8">
    <title>iGEM 2022 ITESO_Guadalajara</title>

    <!-- Mobile Specific Metas
  ================================================== -->
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="description" content="iGEM 2022 ITESO_Guadalajara">
    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=5.0">
    <meta name="author" content="ITESO_Guadalajara">
    <meta name="generator" content="iGEM 2022 ITESO_Guadalajara">
    <!-- Favicon -->
    <link rel="shortcut icon" type="image/x-icon" href="images/favicon.ico" />

    <!-- ** Plugins Needed for the Project ** -->
    <!-- Bootstrap -->
    <link rel="stylesheet" href="plugins/bootstrap/bootstrap.min.css">
    <!-- slick slider -->
    <link rel="stylesheet" href="plugins/slick/slick.css">
    <!-- themefy-icon -->
    <link rel="stylesheet" href="plugins/themify-icons/themify-icons.css">

    <!-- Main Stylesheet -->
    <link href="css/style.css" rel="stylesheet">

</head>

<body>
    <header class="navigation fixed-top">
        <nav class="navbar navbar-expand-lg navbar-dark">
            <a class="navbar-brand font-tertiary h3" href="index.html"><img src="images/logo.png" width="75px"
                    alt="ITESO_Guadalajara"></a>
            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navigation"
                aria-controls="navigation" aria-expanded="false" aria-label="Toggle navigation">
                <span class="navbar-toggler-icon"></span>
            </button>

            <div class="collapse navbar-collapse text-center" id="navigation">
                <ul class="navbar-nav sm-auto">
                    <li class="nav-item active">
                        <a class="nav-link" href="index.html" class="sr-only">Home</a>
                    </li>
                    <li class="nav-item dropdown">
                        <a class="nav-link dropdown-toggle" id="navbarDropdown" role="button" data-toggle="dropdown"
                            href="team-members.html">Team</a>
                        <div class="dropdown-menu" aria-labelledby="navbarDropdown">
                            <a class="dropdown-item" href="team-members.html">Members</a>
                            <div class="dropdown-divider"></div>
                            <a class="dropdown-item" href="team-colabs.html">Collaborations</a>
                            <div class="dropdown-divider"></div>
                            <a class="dropdown-item" href="team-attributions.html">Attributions</a>
                            <div class="dropdown-divider"></div>
                            <a class="dropdown-item" href="team-partnerships.html">Partnerships</a>
                        </div>
                    </li>
                    <li class="nav-item dropdown">
                        <a class="nav-link dropdown-toggle" id="navbarDropdown" role="button" data-toggle="dropdown"
                            href="project-description.html">Project</a>
                        <div class="dropdown-menu" aria-labelledby="navbarDropdown">
                            <a class="dropdown-item" href="project-description.html">Description</a>
                            <div class="dropdown-divider"></div>
                            <a class="dropdown-item" href="project-design.html">Design</a>
                            <div class="dropdown-divider"></div>
                            <a class="dropdown-item" href="project-modeling.html">Model</a>
                            <div class="dropdown-divider"></div>
                            <a class="dropdown-item" href="project-business-model.html">Business model</a>
                            <div class="dropdown-divider"></div>
                            <a class="dropdown-item" href="project-modeling.html">Implementation</a>
                            <div class="dropdown-divider"></div>
                            <a class="dropdown-item" href="project-contributions.html">Contributions</a>
                        </div>
                    </li>
                    <li class="nav-item dropdown">
                        <a class="nav-link dropdown-toggle" id="navbarDropdown" role="button" data-toggle="dropdown"
                            href="#.html">Drylab</a>
                        <div class="dropdown-menu" aria-labelledby="navbarDropdown">
                            <a class="dropdown-item" href="drylab-modeling.html">Modeling</a>
                            <div class="dropdown-divider"></div>
                            <a class="dropdown-item" href="drylab-parts.html">Parts</a>
                            <div class="dropdown-divider"></div>
                            <a class="dropdown-item" href="drylab-engineering-success.html">Engineering success</a>
                            <div class="dropdown-divider"></div>
                            <a class="dropdown-item" href="drylab-protocols.html">Protocols</a>

                        </div>
                    </li>

                    <li class="nav-item dropdown">
                        <a class="nav-link dropdown-toggle" id="navbarDropdown" role="button" data-toggle="dropdown"
                            href="#.html">Human Practices</a>
                        <div class="dropdown-menu" aria-labelledby="navbarDropdown">
                            <a class="dropdown-item" href="human-practices-first-interaction.html">1st interaction</a>
                            <div class="dropdown-divider"></div>
                            <a class="dropdown-item" href="human-practices-first-new-solution.html">1st New Solution</a>
                            <div class="dropdown-divider"></div>
                            <a class="dropdown-item" href="human-practices-second-interaction.html">2nd interaction</a>
                            <div class="dropdown-divider"></div>
                            <a class="dropdown-item" href="human-practices-second-new-solution.html">2nd New
                                Solution</a>
                            <div class="dropdown-divider"></div>
                            <a class="dropdown-item" href="human-practices-third-interaction.html">3rd interaction</a>
                        </div>
                    </li>
                    <li class="nav-item dropdown">
                        <a class="nav-link dropdown-toggle" id="navbarDropdown" role="button" data-toggle="dropdown"
                            href="#.html">Education</a>
                        <div class="dropdown-menu" aria-labelledby="navbarDropdown">
                            <a class="dropdown-item" href="education-amoxtli.html">Amoxtli</a>
                            <div class="dropdown-divider"></div>
                            <a class="dropdown-item" href="education-comunication.html">Comunication</a>
                            <div class="dropdown-divider"></div>
                            <a class="dropdown-item" href="education-Inclusivity.html">Inclusivity</a>
                        </div>
                    </li>
                    <li class="nav-item dropdown">
                        <a class="nav-link dropdown-toggle" id="navbarDropdown" role="button" data-toggle="dropdown"
                            href="#">Awards</a>
                        <div class="dropdown-menu" aria-labelledby="navbarDropdown">
                            <a class="dropdown-item" href="awards-design-league.html">Design League</a>
                            <div class="dropdown-divider"></div>
                            <a class="dropdown-item" href="awards-hackaton.html">Hackathon</a>
                            <div class="dropdown-divider"></div>
                            <a class="dropdown-item" href="awards-cegint.html">CEGINT</a>
                            <div class="dropdown-divider"></div>
                            <a class="dropdown-item" href="awards-igem-epic.html">IGEM EPIC</a>
                        </div>
                    </li>
                </ul>
            </div>
        </nav>
    </header>


    <!-- hero area -->
    <section class="hero-area bg-primary"
        style="background-image: url('/images/parts/Prduccion\ de\ berries-09.jpg') !important; background-size: cover; background-position: center;"
        id="parallax">
        <div class="container" style="text-align: center; background-color: rgba(93,165,243, 0.4)">
            <div class="row">
                <div class="col-lg-12 mx-auto">
                    <h1 class="font-tertiary">Engineering success</h1>
                </div>
            </div>
        </div>


        <!-- social icon -->
        <ul class="list-unstyled ml-5 mt-1 position-relative zindex-1">
            <li class="mb-3"><a class="text-white" href="https://www.facebook.com/iGemITESOGDL/"><i
                        class="ti-facebook"></i></a></li>
            <li class="mb-3"><a class="text-white" href="https://www.instagram.com/igem_iteso/?hl=es-la"><i
                        class="ti-instagram"></i></a></li>
            <li class="mb-3"><a class="text-white" href="https://twitter.com/igemiteso?lang=es"><i
                        class="ti-twitter"></i></a>
            </li>
        </ul>
        <!-- /social icon -->
    </section>
    <!-- /hero area -->

    <!-- team_members -->
    <section class="section">
        <div class="container">
            <div class="row">
                <div class="col-lg-10 mx-auto text-center">
                    <h3>Genetic circuit</h3>
                    <p>After many meetings with our stakeholders, we concluded that the idea which we would develop had
                        to meet the following requirements:
                    </p>

                    <li>
                        Detect and control the infections caused by Fusarium oxysporum.
                    </li>
                    <li>
                        Our product must be safe for cultivation, the environment and the people working with it.
                    </li>
                    <li>
                        If possible, increase the quality of the fruit.

                    </li>
                    <p>
                        Having these criteria in mind and after extensive research we developed BerryVax: a biological
                        system that uses the qualities of Bacillus subtilis as a natural beneficial organism for
                        berries. Our proposal is to stimulate the overexpression of surfactin, one of the main elicitors
                        of B. subtilis which strengthens the plant (Coutte & et al., 2015). Furthermore, BerryVax
                        strives to provide the ability to detect infections caused by F. oxysporum and release
                        antifungal only when this pathogen is detected.
                    </p>
                    <p>
                        In the Labs group, we decided to implement the engineering DBTL cycle (Design-Build-Test-Learn)
                        for the design and construction of our genetic circuit. This was achieved by always having in
                        mind the following three main questions:
                    </p>
                    <li>
                        How can the amount of produced surfactin in B. subtilis be increased?
                    </li>
                    <li>
                        How can F. oxysporum be detected, and which is the most suitable substance to stop the
                        infection by this fungus?
                    </li>
                    <li>
                        How to ensure that BerryVax will not harm the farmers, the environment nor the berry plant
                        itself?
                    </li>

                    <h3>
                        Cycle 1
                    </h3>
                    <h4>DESIGN</h4>
                    <p>
                        As a first prototype, we designed a genetic circuit that consists of two modules that were able
                        to regulate each other like a “switch” (Figure 1.). Module 1 (top of Figure 1): Surfactin, aims
                        to overproduce ilv-leu to promote the metabolic route of leucine and increase the production of
                        biosurfactin (Coutte & et al., 2015) and gtgR, a suppressor of the promotor of Module 2 (bottom
                        of Figure 1): Antifungals, assuring that antifungals will not be produced while Module 1 is
                        active.
                    </p>
                    <p>
                        Additionally, Module 2: Antifungals, was designed with an operon (FAiP) that senses the presence
                        of fusaric acid and acts as a promoter in presence of this metabolite (molecule produced by F.
                        oxysporum). When this module is “on”, GogR will suppress Module 1, AntR will inhibit gtgR and
                        two synthetic antifungals will be expressed: D4E1 (Ryazantsev, 2014) and EcAMP-1 (Rogozhin,
                        2012).
                    </p>
                    <img src="images/parts/F1_Engineering.png" alt="" style="height: 300px;">
                    <p style="text-align: center;">Figure 1. First draft of BerryVax Genetic circuit. </p>
                    <h4>BUILD AND TEST</h4>
                    <p>
                        For this stage we used TinkerCell, a computer-aided design software tool for synthetic biology
                        to simulate the genetic network and the interactions between modules and the F. oxysporum
                        (Figure 2). For simulation purposes, we added the fungus in a simplified form in which it only
                        produces fusaric acid. For this process its expression is inhibited by the antifungals. We also
                        ran the program with the pre-loaded parameters for the Hill equation mathematical model.
                    </p>
                    <img src="images/parts/F2_Engineering.JPG" alt="">
                    <p style="text-align: center;">Figure 2. Genetic Network simulated in TinkerCell.</p>

                    <p>By running the simulation, the graph in Figure 3 was obtained, starting with the presence of
                        fusaric acid. We can observe that the genes of Module 2 increase at the beginning. As the
                        production of antifungals increases, the amount of fusaric acid decreases leading to a very slow
                        drop of gotR. This causes a growth of the expression of Module 1 genes (ilv-leu). However, the
                        presence of antifungals does not seem to be reduced. </p>

                    <img src="images/parts/F3_Engineering.JPG" alt="">
                    <p style="text-align: center;">Figure 3. Simulation of the genetic circuit gene expression in
                        TinkerCell.</p>


                    <h3>ASK</h3>
                    <p>
                        To complement the simulation in TinkerCell, we had a debriefing with our professor Alejandra
                        Barbachano Torres to get feedback and analyze the circuit and the simulation results with her
                        guidance. She pointed that the mechanism of Module 2 for antifungal production was not directly
                        activated for the fusaric acid inducible promotor, causing a delay in the response for the F.
                        oxysporum infection. Moreover, she encouraged us to reconsider two other aspects:
                    </p>
                    <li>
                        Is it necessary for the product to stop the surfactin overproduction when the fusaric acid is
                        detected?
                    </li>

                    <li>
                        Are the synthetic antifungals the best and safest option?
                    </li>

                    <h3>LEARN</h3>
                    <p>
                        Considering the simulation’s results and the feedback of our professor, we concluded that our
                        genetic circuit had to be simplified. We learned that simpler genetic interactions would ease
                        the simulation model thus reducing the chances of having difficulties while assembling our
                        genetic circuit in the lab consequently avoiding the possible impairment of the proper function
                        of BerryVax.
                    </p>

                    <h3>Cycle 2</h3>
                    <h4>DESIGN</h4>
                    <p>
                        We redesigned the circuit considering the learnings from Cycle 1 (Figure 4). In this second
                        attempt we discarded the switch between modules and added Module 3: Killswitch. We selected
                        every part of the circuit carefully to accomplish a specific function that will ensure that
                        everything would work perfectly together. We also tried to use already existing BioBricks that
                        other iGEM Teams were able to use successfully in previous years allowing us to compare our
                        results and even ask for feedback and advice from them if needed. It should also be noted that
                        we changed the synthetic antifungals for biodegradable chitinase as the method to inhibit the
                        fungal infection in the roots of the berry crops.
                    </p>

                    <img src="images/project/F3_Design.png" alt="">
                    <img src="images/project/F4_Design.png" alt="">
                    <img src="images/project/F5_Design.png" alt="" style="height: 300px;">
                    <p style="text-align: center;">Figure 4. BerryVax current genetic circuit.</p>

                    <h3>BUILD</h3>
                    <h4>Construction plan </h4>
                    <p>Once we got a simpler and more functional genetic circuit, we started planning the assembly
                        strategy and the modification of our chassis. After another DBTL process for the assembly
                        strategy and keeping in mind the biosafety aspect of our project, we created a strategy to
                        assemble the three modules of the genetic circuit separately. We will achieve that by using the
                        HiFi Assembly Cloning kit of New England BioLabs, and binding them together in the pUC19 vector
                        with the same kit. The modified vector will then be tested in E. coli competent cells for
                        measuring and modifications of the circuit. </p>

                    <p>Once we validate the genetic circuit in E. coli, we will then insert the circuit into pDG1730, an
                        integrative vector for Bacillus subtilis, ensuring that the plasmid will not transfer by
                        conjugation when BerryVax is ready to be tested on berry plants.
                    </p>
                    <p>Even though we were not able to do the wet lab work due to lack of reactants and logistical
                        issues with our distribution kit and other supplies, we did test our assembly strategy in silico
                        and prepared ourselves for the Wet Lab work (See Assembly Strategy DBTL cycle above).
                    </p>

                    <h3>TEST</h3>
                    <p>We also created a plan to carry out the wet lab work to validate our design with three main
                        measuring protocols to quantify the overproduction of surfactin, to verify the efficiency of the
                        fusaric acid sensor, as well as the kill switch operation and the potential of the chitinase to
                        stop the growth of F. oxysporum. </p>

                    <h3>LEARN</h3>
                    <p>Despite our efforts, we were not able to test our protocols. However, we are aware that in most
                        cases, it is hard to get significant results the first time a protocol is followed, or in our
                        case, the first time we work with a synthetic biology project. So we recommend to check the
                        protocols in our wiki.
                    </p>

                    <h3>ASSEMBLY STRATEGY</h3>
                    <p>
                        Selecting a suitable assembly method and planning a good strategy for it was a challenge for us
                        because this is our first time working with synthetic biology systems. Nonetheless, we managed
                        to design and proof an assembly strategy in silico that joined all our BioBricks together.
                    </p>

                    <h3>Cycle 1</h3>
                    <h4>Design</h4>
                    <p>
                        After extensive research about different assembly methods, we decided the Golden Gate technique
                        was the most suitable for our needs due to the high efficiency and the ability to bind more than
                        12 fragments in one pot. Besides, there are good options for Golden Gate kits that will
                        facilitate the process even more (NEB, 2022). We set a workflow of four assembly reactions: one
                        for the assemble of each module separately and one final reaction to join the three modules to
                        the selected plasmid pUC19. The decision of build the circuit in different reactions instead of
                        every reaction in one pot was made to ensure that the three modules will assemble successfully
                        and to give us the opportunity to verify this step by step with a PCR.
                    </p>

                    <h3>BUILD AND TEST</h3>
                    <p>Once we had all the nucleotide sequences, we used the Codon Optimization Tool of Integrated DNA
                        Technologies (IDT) with our CDS (Coding Sequence) fragments, since some of them have not been
                        used in Bacillus subtilis before (e.g. chitinase A and B). All the gene fragments were approved
                        from the beginning for B. subtilis.
                    </p>

                    <p>
                        Since our plan was to order one of the NEBridge® Golden Gate Assembly Kits, we used the NEBridge
                        Golden Gate Assembly Tool to simulate the process and to design the primers. As a first step, we
                        had to ascertain that the restriction enzymes used in the kits of NEB for Golden Gate would not
                        cut any of our parts. We added the whole genetic circuit in the program and realized that both
                        restriction enzymes SII Bsal and BsmBI will cut our fragments in non-coding regions (in the
                        middle of the regulator region of the fusaric acid inducible promotor) making it impossible to
                        work with that kit.
                    </p>

                    <img src="images/parts/F5_Engineering.png" alt="" style="height: 500px;">
                    <p style="text-align: center;">Figure 5. Results of the analysis of restriction sites for Bsal-HFv2
                        in our sequence using
                        NEBridge Golden Gate Assembly Tool.
                    </p>

                    <h3>LEARN</h3>
                    <p>We learned the importance of analyzing the nucleotide sequence of the genetic circuit and trying
                        to anticipate possible problems in the next stages of wet lab while working in silico.
                        Furthermore, we found interesting the idea of placing specific restriction sites in specific
                        places between parts in case we want to modify our circuit. We also concluded that the best
                        method is not always the newest, but the one that fits the needs of our project better.
                    </p>

                    <h3>Cycle 2</h3>
                    <h4>Design</h4>
                    <p>As a second option, we researched more about the Gibson Assembly method and the HiFi Assembly
                        Cloning Kit of NewEngland BioLabs. This is an option of interest for our genetic circuit as it
                        uses exonucleases instead of endonucleases for the reaction. Moreover, we read about the
                        possibility of binding two double stranded DNA fragments (dsDNA) only with a single stranded DNA
                        oligo (ssDNA) (Figure 6) and we decided to implement it instead of using two primers to add an
                        overlapping region for every part (New England BioLabs, 2016).
                    </p>
                    <img src="images/parts/F6_Engineering.png" alt="" style="height: 400px;">
                    <p style="text-align: center;"> Figure 6. Proposed implementation of restriction sites in specific
                        points of the circuit using
                        HiFi Assembly method.</p>

                    <p>We designed the dsDNA fragments as well as the ssDNA oligos with the NEBuilder Assembly Tool. By
                        using this tool, we made sure that the overlapping section between every fragment was adequate
                        for the assembly reaction. Since we’re planning to synthesize all the fragments, we already
                        included specific restriction site for further modifications to the circuit in Module 1 and 2,
                        taggling the CDS of each module.
                    </p>

                    <h3>BUILD AND TEST</h3>
                    <p>
                        We created a restriction map of each module in order to generate a list of restriction enzymes
                        with no cuts in our design. With this information we made a diagram of the optimal sites for the
                        insertion of specific restriction sites (See the document of Assembly Strategy that you can find
                        in the section “Protocols”).
                    </p>
                    <p>
                        Then, we designed the ssDNA oligos and the dsDNA fragments to bridge all the fragments of each
                        Module. Once we had a successfully assembly every Module in silico, we decided to better use the
                        digest and ligate method to join them together in a single cloning vector by taking advantage of
                        the restriction sites flanking every module.
                    </p>

                    <img src="images/parts/F7_Engineering.png" alt="">
                    <p style="text-align: center;">Figure 7. Diagrams of the process of Assembly a) Assembly of Module 1
                        with HiFi Assembly Cloning Kit, b) Assembly of Module 2 with HiFi Assembly Cloning Kit, c)
                        Assembly of Module 3 with HiFi Assembly Cloning Kit and d) final cloning vector of BerryVax in
                        pUC19 after the standard digest and ligate assembly method.
                    </p>

                    <p>
                        For the reasons mentioned above regarding the wet lab work not being possible to be executed, we
                        did build the fragments and ssDNA oligos as well as the final vector in silico (Figure 7).
                        Finally, we sent our main gene fragments to be synthesized by Twist Bioscience to have them
                        ready for the future steps.
                    </p>

                    <h3>REFERENCES</h3>

                    <p>
                        Coutte, F., Niehren, J., Dhali, D., John, M., Versari, C., & Jacques, P. (2015). Modeling
                        leucine’s metabolic pathway and knockout prediction improving the production of surfactin, a
                        biosurfactant from Bacillus subtilis. Biotechnology Journal, 10(8), 1216–1234.
                        https://doi.org/10.1002/biot.201400541
                    </p>
                    <p>
                        NEB. (2022) Synthetic Biology/DNA Assembly Selection Chart. From
                        https://international.neb.com/tools-and-resources/selection-charts/synthetic-biology-dna-assembly-selection-chart
                    </p>
                    <p>
                        New England BioLabs, (2016) NEBuilder® HiFi DNA Assembly: Bridging dsDNA with a ssDNA Oligo.
                        From:
                        https://international.neb.com/tools-and-resources/video-library/nebuilder-hifi-dna-assembly-bridging-dsdna-with-a-ssdna-oligo
                    </p>
                    <p>
                        Rogozhin, E. (2012). Defense peptides from barnyard grass (Echinochloa crusgalli L.) seeds.
                        Peptides, 38(1), 33–40. https://doi.org/10.1016/j.peptides.2012.08.009
                    </p>
                    <p>
                        Ryazantsev, D. Y. (2014). A novel hairpin-like antimicrobial peptide from barnyard grass
                        (Echinochloa crusgalli L.) seeds: Structure–functional and molecular-genetics characterization.
                        Biochimie, 99, 63–70. https://doi.org/10.1016/j.biochi.2013.11.005
                    </p>


                </div>
            </div>
        </div>
    </section>

    <!-- /blog -->



    <!-- footer -->
    <footer class="bg-dark footer-section">
        <div class="section">
            <div class="container">
                <div class="row">
                    <div class="col-md-4">
                        <h5 class="text-light">Email</h5>
                        <p class="text-white paragraph-lg font-secondary">contactoinfo@contact.com</p>
                    </div>
                    <div class="col-md-4">
                        <h5 class="text-light">Phone</h5>
                        <p class="text-white paragraph-lg font-secondary">+ 00 0000 0000 </p>
                    </div>
                    <div class="col-md-4">
                        <h5 class="text-light">Address</h5>
                        <p class="text-white paragraph-lg font-secondary">Adress Uni</p>
                    </div>
                </div>
            </div>
        </div>
        <div class="border-top text-center border-dark py-5">
            <p class="mb-0 text-light">Owner info</p>
        </div>
    </footer>
    <!-- /footer -->

    <!-- jQuery -->
    <script src="plugins/jQuery/jquery.min.js"></script>
    <!-- Bootstrap JS -->
    <script src="plugins/bootstrap/bootstrap.min.js"></script>
    <!-- slick slider -->
    <script src="plugins/slick/slick.min.js"></script>
    <!-- filter -->
    <script src="plugins/shuffle/shuffle.min.js"></script>

    <!-- Main Script -->
    <script src="js/script.js"></script>

</body>

</html>